pom.xml

```diff
@@ -661,6 +666,11 @@
                 <artifactId>netty-codec-dns</artifactId>
                 <version>${netty.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-transport-native-kqueue</artifactId>
+                <version>${netty.version}</version>
+            </dependency>
```

Comment:
ditto

Reply:
Same here: I noticed that some deps pull in `netty-transport-native-kqueue` but we previously didn't explicitly declare a version override for this artifact. This just pins the version to be in line with all other netty artifacts.